editor/src/clj/editor/lsp/jsonrpc.clj

```diff
@@ -186,7 +202,7 @@
   [{:keys [result error]}]
   (if error
     (throw (ex-info (or (str (:message error))
-                        (case (:code error)
+                        (condp = (:code error)
```

Comment:
Did you do this to get rid of the reflection warning? A better fix would probably be `(case (long (:code error)) ...)`. Although that could throw if `(:code error)` can be `nil`.

Reply:
Yes, I did it to get rid of the reflection warning. We want to throw when there is no code in error — it's required by the JSON-RPC spec.